modules/apps/account/account-service/src/main/java/com/liferay/account/internal/model/listener/AccountEntryUserRelModelListener.java

```diff
@@ -48,6 +51,10 @@ public void onAfterCreate(AccountEntryUserRel accountEntryUserRel)
 		throws ModelListenerException {
 
 		_updateDefaultAccountEntry(accountEntryUserRel.getAccountUserId());
+		_updateAccountMemberRole(
```

Comment:
Account member is an implicit role so we don't want to explicitly assign it to the user. What should be done is add the role only at permission checking, please see https://github.com/pei-jung/liferay-portal/pull/620/commits/8e748cadab99f643c58d5999a09d9fa47bca0780.